editor/src/uuiui/button.tsx

```diff
@@ -36,25 +37,29 @@ export const Button = styled.div<ButtonProps>((props: ButtonProps) => ({
   outline: 'none',
   borderRadius: 1,
   padding: 0,
-  filter: props.spotlight ? 'brightness(90%)' : undefined,
   opacity: props.disabled ? 0.5 : 1,
   pointerEvents: props.disabled ? 'none' : 'initial',
   boxShadow: props.outline ? 'inset 0px 0px 0px 1px hsl(0,0%,90%)' : undefined,
   color: props.primary ? 'white' : 'inherit',
-  background: props.primary ? UtopiaStyles.backgrounds.blue : 'transparent',
+  background: props.primary
+    ? UtopiaStyles.backgrounds.blue
+    : props.spotlight
+    ? UtopiaTheme.color.secondaryBackground.value
+    : undefined,
```

Comment:
Nested ternaries?! You monsters!

Reply:
@OmarDaSilva this is my doing, I'll fix it. We have a coding convention against nested ternaries which I always ~choose to~ forget :)

Reply:
#ace17cf

Reply:
it is still not fixed, and #ace17cf is not highlighted by github – was a commit lost somewhere?